Code/Single_Pendulum/single_increase_parametric.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, values, all_data, **kwargs):
         self.prev_time, self.curr_time = 0.0, values['time']
         self.moving_down = True
 
-        self.max_times = self.last_maxima(all_data)
+        self.max_times = self.last_maxima(all_data['time'])
```

Comment:
won't this break it? Think it is missing another list that needs to be passed in.